### DISCUSSION TITLE
Don't overwrite the weekly alpha release.

### DISCUSSION THREAD

**Discussion by ndgnuh** [gama-platform/gama#109] [Ideas]
Currently, when a new weekly alpha build is out, the old one is overwritten.
![image](https://user-images.githubusercontent.com/33273948/115351520-8cafa680-a1e0-11eb-87be-131f5ae901a8.png)

I suggest adding a commit hash after release name and do a separate release (like the one of the image below). This way it will be easier to track the problems (if there's any) and revert to the working version.
![image](https://user-images.githubusercontent.com/33273948/115351742-c2ed2600-a1e0-11eb-87a9-17cdfdd3d5d5.png)

Or use release date
![image](https://user-images.githubusercontent.com/33273948/115351866-e9ab5c80-a1e0-11eb-9970-43952c122849.png)

**Comment by hqnghi88** [comment]
the name of the packages do the job, however we can add as your suggestion, the suffix into the name of release

**Reply by ndgnuh** [reply]
I mean that we create a new release, and don't overwrite the old one. There would be `Alpha Version 1.8.2_thisweek` and `Alpha version 1.8.2_nextweek` and so on. If something does happens, users can just download the latest working version.

**Reply by hqnghi88** [reply]
ok, NO for that, for many reason and we dont create such kind of release.

**Reply by ndgnuh** [reply]
Can you explain why?

**Reply by AlexisDrogoul** [reply]
Hi, the main reason is that the activity in terms of release is really important during intense coding periods (and we are going to enter one with GAMA 1.8.2), during which it doesn't really make sense to ask users to download GAMA every week ad there might be very big bugs and issues. The ones really interested in following are usually the developers themselves (and they know how to build GAMA from source). Otherwise, the differences between two weekly versions is too small to justify keeping the old ones. Last reason may seem less important but is nevertheless something we will need to pay attention to in the future: keeping old versions is a waste of resources and energy and we should try (collectively of course) to be more frugal in terms of digital resources.
